app/gui/src/project-view/components/GraphEditor/widgets/WidgetFunction/widgetFunctionCallInfo.ts

```diff
@@ -111,11 +112,14 @@ export function useWidgetFunctionCallInfo(
       Ast.TextLiteral.new(JSON.stringify(args)).code(),
     ]
 
+    const modulePath = project.moduleProjectPath.value
```

Comment:
I need a _TypeScript expert hint_:
- `error TS2339: Property 'value' does not exist on type 'ProjectPath'.`
- as reported by [typescript check](https://github.com/enso-org/enso/actions/runs/15867231650/job/44737688018?pr=13342#step:9:25)
- but without `.value` I was not getting the right behavior

Any hint? What shall be done about it?

Reply:
Thanks. I was surprised the problem only manifests in VSCode text editor, not from a command line. When I do:
```
enso$ corepack pnpm compile
```
it yields, no error! Hence I was suspecting the VSCode editor is broken (as is often the case with Java and Scala)...

Reply:
Changed in 57ff814

Reply:
> When I do:
> 
> ```
> enso$ corepack pnpm compile
> ```
> 
> it yields no error! 

Yes, `compile` command compiles libraries only, and the error was in the application.